lib/public/views/Logs/Create/logCreationComponent.js

```diff
@@ -46,7 +46,7 @@ export const logCreationComponent = (creationModel) => {
  */
 const logCreationForm = (creationModel) => [
     h('.w-100.flex-row.mv2.justify-between', [
-        h('h3', 'New log'),
+        h('h3', `${creationModel.logType}`),
```

Comment:
Supposing you have the parent log as a parameter, you can use here:
``${parentLog ? `Reply to log: ${parentLog.title}` : 'New log'}``